### PR TITLE
Add 2023 Kentucky Tax Rules

### LINKED ISSUE #4177: Add 2023 Kentucky Tax Rules
2023 Income Tax Form Instruction: https://revenue.ky.gov/Forms/740%20Packet%20Instructions%202023.pdf

### PR DESCRIPTION
**Pull request by vrathi101**
Fixes #4177

Changes: 
- increase in standard deduction to 2980
- decrease in individual income tax rate to 4.5%

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ky/tax/income/exclusions/pension_income/threshold.yaml [hunk 1]
```diff
@@ -30,5 +30,7 @@ metadata:
       href: https://revenue.ky.gov/Forms/Schedule%20P%202018.pdf
     - title: Kentucky Pension Income Schedule P (2022)
       href: https://revenue.ky.gov/Forms/Schedule%20P%202022.pdf
+    - title: 2023 Kentucky Pension Income Exclusion (Schedule P)
+      href: https://revenue.ky.gov/Forms/Schedule%20P%202023.pdf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ky/tax/income/exclusions/pension_income/threshold.yaml:34]: add pdf page

File: policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml [hunk 2]
```diff
@@ -10,7 +11,9 @@ metadata:
   # Legal code specifies $1,700 in 1999, CPI-inflated since.
   - title: Kentucky Revised Statutes, KRS Chapter 141.081 Optional standard deduction for individuals - Exception.
     href: https://apps.legislature.ky.gov/law/statutes/statute.aspx?id=29067
-  - title: 2022 KENTUCKY Form 2022
+  - title: 2023 KENTUCKY Form 740
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml:14]: is this adjusted with inflation?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml [hunk 3]
```diff
@@ -2,15 +2,19 @@ description: Kentucky provides this standard deduction amount.
 values:
   2021-01-01: 2_690
   2022-01-01: 2_770
+  2023-01-01: 2_980
 metadata:
+  uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml:7]: is this rounded? Need to check legal code, if so, we will need to add some extra uprating metadata:
e.g.

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml (hunk 3 above)
Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml:7]: please resolve this

Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml:7]: ok please leave a comment mentioning that

**Review by MaxGhenis — APPROVED** [review]
good to merge after comment on uprating rounding

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**